packages/manager/apps/dedicated/client/app/components/dedicated-cloud/datacenter/vmware-vdc-add/dedicatedCloud-vmware-vdc-add.constants.js

```diff
@@ -0,0 +1,20 @@
+export const DATASTORES_REQUIRED = 2;
+
+export const HOST_TYPE_MAP = [null, 'vsan', 'saphana'];
```

Comment:
I created an object export const HOST_TYPE = {
  STANDARD: 'standard',
  VSAN: 'vsan',
  SAPHANA: 'saphana',
};

and now i use Object.values for apply you solution. I use Standard because this is the name of the type in the content.